src/Cake.Svn.Tests/packages.config

```diff
@@ -60,4 +60,5 @@
   <package id="xunit.extensibility.core" version="2.3.0" targetFramework="net46" />
   <package id="xunit.extensibility.execution" version="2.3.0" targetFramework="net46" />
   <package id="xunit.runner.visualstudio" version="2.3.0" targetFramework="net46" developmentDependency="true" />
+  <package id="SharpSvn.1.9-x64" version="1.9007.3987.251" targetFramework="net46" />
```

Comment:
This has the drawback that we no longer can check if we completely abstract SharpSVN API. But otherwise we cannot add testcases for internal stuff. If we wan't to have both we need to split up the test classes, but OK for me to keept it like this for the moment.